collections/Douce/MS_Douce_104.xml

```diff
@@ -37,40 +36,238 @@
+                        </list>
+                     </decoNote>
+                     <decoNote type="colInit">3-line decorated initials at passus beginnings, blue with red background.</decoNote>
+                     <decoNote type="rubrication">Passus headings, Latin words, and some names underlined in red.</decoNote>
                   </decoDesc>
+                  <bindingDesc>
+                     <binding notAfter="1629" notBefore="1600">
+                        <p>Brown tanned calf over laminated pulpboard, early seventeenth century; with gilt armorial stamp of James Ley and double gilt fillet. Red leather spine with gilt lettering. Removed and kept separately within the box.</p>
+                     </binding>
+                     <binding when="1987">
+                        <p>Rebound, 1987.</p>
+                     </binding>
+                  </bindingDesc>
                </physDesc>
                <history>
                   <origin>
-                     <origDate calendar="Gregorian" when="1427" cert="high">1427</origDate>
-                     <origPlace cert="low">
-                        <country key="place_1000078">Irish</country> (?)</origPlace>
+                     <origDate calendar="Gregorian" cert="high" when="1427-09-25">Colophon dating the start of the manuscript’s creation 25 September 1427, with it evidently being completed in 1427–28 (fol. 112v).</origDate>
+                     <origPlace><country key="place_1000078">Ireland</country>, The Pale (?)</origPlace>
+                     <p>Suggested to be written in the dialect spoken by English settlers in Ireland by Angus McIntosh, M. L. Samuels, and Michael Benskin, <title>A Linguistic Atlas of Late Mediaeval English</title> (Aberdeen, 1986), 1:147, based on analysis from fols. 50r-79v, with a few additions from text to fol. 108v. Irish scribes were also active in London, but this possibility seems less likely based on the artistic style and text: see Pearsall, p. xiii.</p>
                   </origin>
-                  <provenance notAfter="1834" notBefore="1757" resp="#MMM">
-                     <persName key="person_69848690" role="fmo">Francis Douce, 1757–1834</persName>
-                  </provenance>
+                  <provenance><persName key="person_71740714" role="fmo">James Ley, first earl of Marlborough (1550–1629)</persName>, chief justice of king’s bench in Ireland: his arms on the former binding.</provenance>
+                  <provenance notAfter="1834" notBefore="1757" resp="#MMM"><persName key="person_69848690" role="fmo">Francis Douce (1757–1834)</persName>: his bookplate, notes (upper pastedown and fol. 33a), and sporadic line numbering.</provenance>
```

Comment:
details on Douce's acquisition in Douce Legacy https://babel.hathitrust.org/cgi/pt?id=mdp.39015036877796&view=1up&seq=168&skin=2021